.buildkite/steps/release.sh

```diff
@@ -51,10 +51,8 @@ agent_digest=$(crane digest "ghcr.io/buildkite/agent-stack-k8s/agent:$version")
 # TODO: remove once world write issues is fixed
 git stash -uk
 
-goreleaser release \
-  --clean \
-  --release-notes <(ghch --format=markdown --from="$previous_tag" --next-version="$tag") \
-  --release-footer <(cat <<EOF
+changelog=$(ghch --format=markdown --from="$previous_tag" --next-version="$tag")
+footer="
```

Comment:
I wondered how to keep this as a heredoc; the answer is something like

```shell
read -r -d '' footer <<EOF
...
EOF
```

Reply:
Nice, taht's a good one to have up one's sleeve.